packages/flutter_tools/lib/src/test/runner.dart

```diff
@@ -63,6 +65,10 @@ Future<int> runTests(
     '--concurrency=$concurrency',
     for (final String name in names)
       ...<String>['--name', name],
+    if (tags != null && tags.isNotEmpty)
+      ...<String>['--tags', tags],
+    if (excludeTags != null && tags.isNotEmpty)
```

Comment:
Should this line be `if (excludeTags != null && excludeTags.isNotEmpty)`? If so, consider adding a test that would have caught this.